src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx

```diff
@@ -37,30 +49,51 @@ export const ItemFlow = ({ name, index, onRemove }: ItemFlowProps) => {
   const { error } = getFieldState(itemName);
 
   const title = (
-    <StyledTitle component="span" sx={{ position: 'relative' }}>
+    <StyledTitle component="span">
       {error && <Badge variant="dot" color="error" />}
       {t('activityItemsFlowItemTitle', { index: index + 1 })}
     </StyledTitle>
   );
+  const observerSx = { position: 'absolute', height: 'calc(100% + 20rem)', bottom: 0 };
```

Comment:
can be moved to ItemFlow.const.ts
maybe it's better to rename to observerStyles

Reply:
done